utils/theme.ts

```diff
@@ -3,87 +3,89 @@ import { css } from '@emotion/react';
 
 export const globalStyles = css`
   body {
-    font-family: 'Orbitron', sans-serif;
+    font-family: 'Unbounded', sans-serif;
     font-size: 1rem;
-    background: #fff;
-    color: #000;
+    background: #171922;
+    color: #fff;
   }
 `;
 
 const Button = {
+  baseStyle: {
+    _active: {
+      background: '#2D3748',
+      transform: 'scale(0.98)',
+      _disabled: {
+        transform: 'scale(1)',
+      },
+    },
+  },
   defaultProps: {
     variant: 'outline',
+    // _hover: {
+    //   transform: 'scale(1.1)',
+
+    // },
   },
   sizes: {
-    sm: {
+    xs: {
       fontSize: '12px',
-      w: '100px',
+      h: 'auto',
+      pt: '2',
+      pb: '2',
+      px: '6',
+    },
+    sm: {
+      fontSize: '14px',
+      h: 'auto',
+      pt: '2',
+      pb: '2',
+      px: '6',
     },
     md: {
       fontSize: '16px',
-      h: '50px',
-      w: '180px',
+      lineHeight: '20px',
+      h: 'auto',
+      pt: '2',
```

Comment:
Should this be `pt: 3`? Looks slightly off-center:

<img width="242" alt="Screenshot 2023-10-11 at 3 24 45 PM" src="https://github.com/raid-guild/character-sheets-frontend/assets/40322776/326b522a-79b5-4ed9-8577-832a11e4433b">

Reply:
Yeah again an issue related to the font not loading. See below, the button with Unbounded, seems to fix the off center thing? Or what do you think @ECWireless ?
<img width="369" alt="Screenshot 2023-10-12 at 09 39 27" src="https://github.com/raid-guild/character-sheets-frontend/assets/51742196/ea5a13a1-aad0-42f6-a724-f0719dca75f7">